src/mono/wasm/runtime/startup.ts

```diff
@@ -334,6 +334,8 @@ async function postRunAsync(userpostRun: (() => void)[]) {
         Module["FS_createPath"]("/", "usr", true, true);
         Module["FS_createPath"]("/", "usr/share", true, true);
 
+        if (MonoWasmThreads) tcwraps.mono_wasm_init_finalizer_thread();
```

Comment:
would prefer the call to be on another line, we don't do single-line `if (x) y;` often in this codebase